src/sqlfluff/utils/reflow/elements.py

```diff
@@ -37,61 +37,76 @@ def _class_types(segments: Sequence[RawSegment]) -> Set[str]:
         return set(chain.from_iterable(seg.class_types for seg in segments))
 
     @property
-    def class_types(self):
-        """The set of contained class types.
+    def class_types(self) -> Set[str]:
+        """Get the set of contained class types.
 
-        Parallel to BaseSegment.class_types
+        Parallel to `BaseSegment.class_types`
         """
         return self._class_types(self.segments)
 
     @property
-    def raw(self):
+    def raw(self) -> str:
         """Get the current raw representation."""
         return "".join(seg.raw for seg in self.segments)
 
     def num_newlines(self) -> int:
-        """How many newlines does this element contain?"""
+        """Count the number of newlines this element contains."""
         return sum(bool("newline" in seg.class_types) for seg in self.segments)
 
 
 @dataclass(frozen=True)
 class ReflowBlock(ReflowElement):
     """Class for keeping track of elements to reflow.
 
+    This class, and it's sibling :obj:`ReflowPoint`, should not
+    normally be manipulated directly by rules, but instead should
+    normally be manipulated only via the :obj:`ReflowSequence`.
+
     It holds segments to reflow and also exposes configuration
-    around how they are expected to reflow around others.
+    around how they are expected to reflow around others. Typically
```

Comment:
Avoids using "around" twice in close proximity
```suggestion
    regarding how they are expected to reflow around others. Typically
```